microsoft-edge/webview2/how-to/winrt-from-js.md

```diff
@@ -0,0 +1,366 @@
+
+1. Click the **Install** button.
+
+WIL is now installed for the **WinRTAdapter** project.  Windows Implementation Library (WIL) is a header-only C++ library to make using COM coding for Windows easier.  It provides readable, type-safe C++ interfaces for Windows COM coding patterns.
+
+
+<!-- =============================================== -->
+## Step 4. Install WebView2 prerelease SDK, for WinRTAdapter project
+
+In the WinRTAdapter project, also install a prerelease version of the WebView2 SDK, as follows:
+
+1. In Solution Explorer, right-click the **WinRTAdapter** project, and then select **Manage NuGet Packages**.  The NuGet Package Manager window opens.
+
+1. Select the **Include prerelease** checkbox.
+
+1. In the **Search** box, enter **WebView2**.
+
+1. Click the **Microsoft.Web.WebView2** card.  Detailed info appears in the middle area of the window.
+
+1. In the **Version** drop-down, select a **prerelease** version of the WebView2 SDK.  The version must be 1.0.1243.0 or higher.  Note which version number you select.
+
+1. Click the **Install** button:
+
+   ![NuGet Package Manager, selecting the WebView2 SDK package, for the WinRTAdapter project.](winrt-from-js-images/pkg-mgr-wv2-sdk.png)
+
+The WebView2 prerelease SDK is now installed for the **WinRTAdapter** project.
+
+
+<!-- =============================================== -->
+## Step 5. Install WebView2 prerelease SDK, for webview2_sample_uwp project
+
+In the **webview2_sample_uwp** project, install the same prerelease version of the WebView2 SDK as you installed for the **WinRTAdapter** project, as follows:
+
+1. In Solution Explorer, right-click the **webview2_sample_uwp** project, and then select **Manage NuGet Packages**.  The NuGet Package Manager window opens.
+
+1. Select the **Include prerelease** checkbox.
+
+1. In the **Search** box, enter **WebView2**.
+
+1. Click the **Microsoft.Web.WebView2** card.  Detailed information appears in the middle area of the window.
+
+1. In the **Version** drop-down, select a **prerelease** version of the WebView2 SDK.  The version must be 1.0.1243.0 or higher.  This needs to be the same version as for the **WinRTAdapter** project.
+
+1. Click the **Install** button.
+
+   The screen should look similar to the above section, except that now, **NuGet Package Manager** is open for the **webview2_sample_uwp** project instead of the **WinRTAdapter** project.
+
+The WebView2 prerelease SDK is now installed for the **webview2_sample_uwp** project.
+
+
+<!-- =============================================== -->
+## Step 6. Generate source code for selected host API items
+
+Next, configure the wv2winrt tool (the WebView2 WinRT JS Projection tool), to incorporate the WinRT classes that you want to use.  This generates source files that will then be compiled, rather than adding a `#` include of entire header files.  Generating code for these API items enables your web-side JavaScript code to call these API items.
```

Comment:
The 'rather than adding a `#` include of entire header files' seems to maybe be referencing how C++/WinRT works? Without context it seems strangely out of place here and maybe is OK to remove.